internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -749,6 +749,15 @@ components:
         region:
           type: string
           example: 'eu-west-1'
+        public:
+          type: boolean
+          default: false
+          description: |
+            By default, a long, obfuscated URL is returned. Its expiration
```

Comment:
I was a bit confused by this, maybe `If false, the default, a long ..."

Reply:
it would make it a little clearer. Otherwise lgtm

Reply:
I reworded the description slightly.